### DISCUSSION TITLE
err_connection_aborted and err_connection_reset

### DISCUSSION THREAD

**Discussion by mikered987** [cefsharp/CefSharp#4315] [General]
I have two computer. On one the CefSharp minimal example works only once. If starting it a second time it constantly produces the error:
_Failed to load URL http://www.google.com/ with error ERR_CONNECTION_ABORTED (ConnectionAborted)._
or:
_Failed to load URL https://www.google.com/ with error ERR_CONNECTION_RESET (ConnectionReset)._
On the second computer it works fine.
Is there something which changes after the first start? Rebuild of the project doesn't help.
Or is there some kind of timeout value or similar which could be increased? 
Normal browser work fine on that pc.

**Comment by amaitland** [comment]
What's different between the two machines? Firewall? Antivirus? Security software? Etc

You'll need to do a thourough comparison. 

Check the log file for errors see https://github.com/cefsharp/CefSharp/wiki/Trouble-Shooting#log-file

> Is there something which changes after the first start?

The cache doesn't exist on first start. You can try deleting it for testing purposes. 

> Or is there some kind of timeout value or similar which could be increased?

Stay away from playing with settings

**Reply by mikered987** [reply]
> What's different between the two machines? Firewall? Antivirus? Security software? Etc

For software they are mostly the same. Same Windows, same antivirus and firewall. The one with the issue has additionally arconis backup software installed.
Main difference beside that is hardware and thus the drivers. 

> Check the log file for errors see https://github.com/cefsharp/CefSharp/wiki/Trouble-Shooting#log-file

It adds only two lines to the debug.log file:
[1128/000849.605:ERROR:crashpad_client_win.cc(534)] CreateProcess: The system cannot find the file specified. (0x2)
[1128/000849.702:INFO:crash_reporting.cc(211)] Crash reporting enabled for process: browser

> The cache doesn't exist on first start. You can try deleting it for testing purposes.

Deleting the cache didn't change it.

**Reply by amaitland** [reply]
> Main difference beside that is hardware and thus the drivers.

In my experience this sort of problem is software, not hardware/drivers.

> It adds only two lines to the debug.log file:

How does this compare to the working machine? 

Try running in a virtual machine or windows sandbox on the problematic machine. Making sure to install Microsoft Visual C++

**Reply by mikered987** [reply]
> How does this compare to the working machine?

They are the same.

> Try running in a virtual machine or windows sandbox on the problematic machine. Making sure to install Microsoft Visual C++

Via sandbox it works.
